File: dl-invites/app/src/main/java/com/google/firebase/dynamicinvites/presenter/EmailPresenter.java
package com.google.firebase.dynamicinvites.presenter;

import android.content.Context;
import android.widget.Toast;

import com.google.firebase.dynamicinvites.R;
import com.google.firebase.dynamicinvites.kotlin.model.InviteContent;

public class EmailPresenter extends InvitePresenter {

    public EmailPresenter(boolean isAvailable, InviteContent content) {
        super("Email", R.drawable.ic_email, isAvailable, content);
    }

    @Override
    public void sendInvite(Context context) {
        super.sendInvite(context);
        Toast.makeText(context, "TODO: Implement email sending", Toast.LENGTH_SHORT).show();
    }
}

File: dl-invites/app/src/main/java/com/google/firebase/dynamicinvites/kotlin/presenter/EmailPresenter.kt
package com.google.firebase.dynamicinvites.kotlin.presenter

import android.content.Context
import android.widget.Toast

import com.google.firebase.dynamicinvites.R
import com.google.firebase.dynamicinvites.kotlin.model.InviteContent

class EmailPresenter(isAvailable: Boolean, content: InviteContent) :
    InvitePresenter("Email", R.drawable.ic_email, isAvailable, content) {

    override fun sendInvite(context: Context) {
        super.sendInvite(context)
        Toast.makeText(context, "TODO: Implement email sending", Toast.LENGTH_SHORT).show()
    }
}
